src/app/components/Forms/PopupForm.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import styled from 'styled-components'
+import Button from '../Button/Button'
+import OverlayWrapper from '../OverlayWrapper/OverlayWrapper'
+
+type FormProps = {
+  onCancel: () => void
+  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void
+  SubmitText: string
+  children: React.ReactNode
+}
+
+export default function Form({
+  onCancel,
+  onSubmit,
+  children,
+  SubmitText,
+}: FormProps): JSX.Element {
+  return (
+    <OverlayWrapper onReturn={onCancel}>
+      <FormContainer onSubmit={onSubmit}>
```

Comment:
You could think about adding ` event.preventDefault()` here. You only need it once afterwards.